src/components/asset.tsx

```diff
@@ -24,6 +24,22 @@ const Asset: React.FC<{
 
   const aspectRatio = block_aspect_ratio || block_height / block_width;
 
+  if (type === "figma") {
+    return (
+      <div
+        style={{
+          paddingBottom: `${aspectRatio * 100}%`,
+          position: "relative"
+        }}
+      >
+        <iframe
+          className="notion-image-inset"
+          src={value.properties.source[0][0]}
```

Comment:
can we move this into `(type === "embed" || type === "video")` and add `|| type === "figma"`

and then:

```jsx
<iframe
          className="notion-image-inset"
          src={type === "figma" ? value.properties.source[0][0] : display_source}
/>
```

we can also improve the if with 
`["embed","video","figma"].includes(type)`

Reply:
Sure!